catalog/USGS/USGS_NLCD_RELEASES_2021_REL_TCC_v2021-4.jsonnet

```diff
@@ -0,0 +1,317 @@
+    LandTrendr - Temporal segmentation algorithms. In Remote Sensing of Environment.
+    *Science Direct*,
+    114(12): 2897-2910
+    [doi:10.1016/j.rse.2010.07.008](https://doi.org/10.1016/j.rse.2010.07.008)
+
+    * **Kennedy, R., Yang, Z., Gorelick, N., Braaten, J., Cavalcante, L., Cohen, W., and Healey, S., 2018.**
+    Implementation of the LandTrendr Algorithm on Google Earth Engine. In Remote Sensing. *MDPI*,
+    10(5): 691
+    [doi:10.3390/rs10050691](https://doi.org/10.3390/rs10050691)
+
+    * **Sentinel-Hub, 2021.**
+    Sentinel 2 Cloud Detector. [Online].
+    Available at: [https://github.com/sentinel-hub/sentinel2-cloud-detector](https://github.com/sentinel-hub/sentinel2-cloud-detector)
+
+    * **Zhu, Z., and Woodcock, C. E., 2012.**. 
+    Object-based cloud and cloud shadow detection in Landsat imagery. In Remote Sensing of
+    Environment. *Science Direct*, 
+    118: 83-94
+    [doi:10.1016/j.rse.2011.10.028](https://doi.org/10.1016/j.rse.2011.10.028)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license('https://data.fs.usda.gov/geodata/rastergateway/treecanopycover/')
+  ],
+  keywords: [
+    'forest',
+    'gtac',
+    'landsat_derived',
+    'redcastle_resources',
+    'sentinel2_derived',
+    'time_series',
+    'usda',
+    'usfs',
+    'usgs',
+  ],
+  providers: [
+    ee.producer_provider('USDA Forest Service (USFS) Geospatial Technology and Applications Center (GTAC)', 
+    'https://www.fs.usda.gov/about-agency/gtac'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(-135.286387, 20.38379, -56.446306, 52.459364,
+                    '2008-06-01T00:00:00Z', '2021-09-30T00:00:00Z'),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'study_area',
+        description: |||
+          TCC currently covers CONUS, Southeastern Alaska, Puerto Rico-US
+          Virgin Islands and Hawaii. This version contains all study areas.
+          Possible values: 'CONUS, SEAK, PRUSVI, HI'
+        |||,
+        type: ee_const.var_type.string,
+      },
+    ],
+    gsd: [30],
+    'eo:bands': [
+      {
+        name: 'Science_Percent_Tree_Canopy_Cover',
+        description: |||
+          The raw direct model outputs. Each pixel has a mean predicted tree canopy 
+          cover value for each year.
+        |||,
+      },
+      {
+        name: 'Science_Percent_Tree_Canopy_Cover_Standard_Error',
+        description: |||
+          The standard deviation of the predicted values from all regression trees we refer 
+          to as standard error. Each pixel has a standard error for each year.
+        |||,
+      },
+      {
+        name: 'NLCD_Percent_Tree_Canopy_Cover',
+        description: |||
+          To produce NLCD tree canopy cover, a post-processing workflow is applied to the direct 
+          model output that identifies and sets non-treed pixel values to zero percent 
+          tree canopy cover.
```

Comment:
please say it's fully masked-out in some years

Reply:
Thanks for catching this. I have added text